### ISSUE TITLE
AccessToken declined

### ISSUE THREAD

**Issue by Guest007** [twilio/twilio-python#388]
**Version:** 6.5.2

### Code Snippet
```python
ipm_grant = ChatGrant(
                    service_sid="IS3308....."
                )
token = AccessToken(
                account="ACfc......",
                key_sid="SKf5b.....",
                key_secret="pbv3x......",
                [ipm_grant],
                identity="Dsp:4:7",
                ttl=600
            ).to_jwt()
```

Got well token. But:

### Exception/Log
Client's requests (from JS and Android's actual tools) returns
```
{"message":"The authorization with Token failed","code":2016,"user_error":false,"params":{"description":"Could not authenticate fpa token"}}
```
Couple of days before all worked. 
Where can be a trouble? Some changes in API? Wrong AccessToken creation?

**Comment by efossier** [comment]
@Guest007 Im sorry you're having trouble, I will make some time today to investigate this. 

Can you clarify on what you meant by it was all working a couple of days ago? Was your python code the same? Did you upgrade your library? Or just suddenly started failing (would indicate a change in the API)?

**Comment by Guest007** [comment]
Thank you

> Can you clarify on what you meant by it was all working a couple of days ago?
08-18-2017 it definitely worked. We are faced with problems 08-20-2017

> Was your python code the same?
In monday I tried to understand. I found out that I don't need to point `endpoint_id` in `ChatGrant`. 

>  Did you upgrade your library?
And I update library from 6.5.1 to 6.5.2

That's all.

Versions of client's libraries (JS and Android) are freshes.

**Comment by efossier** [comment]
@Guest007 Just an update, I have not found the root cause yet however I've reached out to some teams internally to see if anything changed on their end between 8-18 and 8-20.

On our end nothing changed related to Jwts in 6.5.1 to 6.5.2. I apologize for the inconvenience this is causing you and we are doing our best to track down the regression. 

In the meantime have you tried reverting to older versions of the python, js, android libraries to confirm this is an issue on the api side?

**Comment by Guest007** [comment]
Hi
We are found a point of trouble. This is `nbf` parameter in JWT. By the way test tokens, generated from Twilio's "Testing Tools" do not contain this parameter. It's optional. 
I tried to synchronize server's time with ntp, try to set offset for `nbf` - it does not help. Only if I remove this from JWT-token
```Python
token = AccessToken(
                account="ACfc......",
...
            )
del token.nbf
token = token.to_jwt()
```
But this is needed little hack in https://github.com/twilio/twilio-python/blob/c832931fc686bc9b1dcc945a84457da7056d9162/twilio/jwt/__init__.py#L83
and transform it in something like that:
```Python
if 'nbf' in payload:
    payload['nbf'] = self.nbf or int(time.time())
```

May be trouble in twilio's server that approving tokens (time delta)?

But it not such a rare problem: https://stackoverflow.com/a/37811754/1029518

May be use `nbf` only of it directly set by user?

Thank you

**Comment by efossier** [comment]
@Guest007 Ah good catch, yeah we can make that change.

**Comment by efossier** [comment]
https://github.com/twilio/twilio-python/pull/390

**Comment by efossier** [comment]
We shipped the PR above in `6.6.0`, you can disable setting `nbf` on jwts by passing `nbf=None` in the constructor.
